frontend/scripts/react-components/home/entrypoints.component.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import Siema from 'react-siema';
+import Link from 'redux-first-router-link';
+import debounce from 'lodash/debounce';
+
+class Entrypoints extends React.PureComponent {
+  static getIsMobile() {
+    return window.innerWidth <= 700;
+  }
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentSlide: 0,
+      isMobile: Entrypoints.getIsMobile()
+    };
+    this.entrypoints = [
+      {
+        link: { type: 'profileRoot' },
+        subtitle: 'Profile',
+        text:
+          'View the trade and sustainability profile of a particular' +
+          ' company or production region.',
+        className: '-start'
+      },
+      {
+        link: { type: 'tool' },
+        subtitle: 'Supply Chain',
+        text:
+          'Follow trade flows to identify sourcing regions, profile supply chain risks and' +
+          ' assess opportunities for sustainable production.',
+        className: '-mid'
+      },
+      {
+        link: { type: 'tool', payload: { state: { isMapVisible: true } } },
+        subtitle: 'Map',
+        text:
+          'Explore the sustainability of different production regions and identify risks and' +
+          ' opportunities facing downstream buyers.',
+        className: '-end'
+      }
+    ];
+    this.getSliderRef = this.getSliderRef.bind(this);
+    this.onClickPrev = this.onClickPrev.bind(this);
+    this.onClickNext = this.onClickNext.bind(this);
+    this.onResize = debounce(this.onResize.bind(this), 300);
+    this.renderEntrypoints = this.renderEntrypoints.bind(this);
```

Comment:
we don't actually need those two render functions bindings.

Reply:
We do cause we’re accessing this.state and this.entrypoints